core/templates/dev/head/pages/exploration_player/AudioBarDirective.js

```diff
@@ -39,7 +39,7 @@ oppia.directive('audioBar', [
             WindowDimensionsService, AudioTranslationManagerService,
             EVENT_AUTOPLAY_AUDIO, BrowserCheckerService) {
           $scope.audioBarIsExpanded = false;
-          $scope.hasPressedPlayButtonAtLeastOnce = false;
+          $scope.showProgressBar = true;
```

Comment:
@import-keshav, by default, this should be false right? (have you compared your UI changes with the develop branch?)

Reply:
Yes sir, i compared it.
by default progress bar is enable.

Reply:
I don't any progress bar after opening/sliding the audio bar on production.
![image](https://user-images.githubusercontent.com/16653571/52905719-2aeffc80-3264-11e9-8067-67587ae410e1.png)

[Link to exploration.](https://www.oppia.org/explore/MjZzEVOG47_1?collection_id=4UgTQUc1tala)

Reply:
Ahh sorri